instaloader/structures.py

```diff
@@ -876,6 +876,20 @@ def business_category_name(self) -> str:
     def biography(self) -> str:
         return self._metadata('biography')
 
+    def get_sponsors(self) -> List['Profile']:
+        """
+        Return a list of :class:`Profile` sponsors wich you can iterate over.
+
+        .. versionadded:: 4.10
```

Comment:
We released `4.10`, this should go into `4.11` then.

```suggestion
        .. versionadded:: 4.11
```